src/ansible_creator/resources/common/patterns/sample_pattern/playbooks/site.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Weather forecast
+  hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: Ensure the airport code has been defined
```

Comment:
Can we show an inline argspec validation here?  Since it's small, the argspec can just be a var for the task.

Reply:
Done!